source/documentation/partials/_messagetypeintro.md

```diff
@@ -1,4 +1,20 @@
-## Accessing the XML Schemas
+## Message details
+
+This section provides the following information about each message type: 
+
+- Characteristics of the data groups belonging to the message: sequence, number of repetitions, and a status value to indicate if the data group is: 
+    - mandatory (R: Required), 
+    - optional (O: Optional) or 
+    - conditional (D: Dependent).
+- Characteristics of the data items belonging to a data group: sequence, number of repetitions, type, length, and a value to indicate if a data item is: 
+    - mandatory (R: Required), 
+    - optional (O: Optional) or 
+    - conditional (D: Dependent).
+- Data group nesting is shown with dashes, which means that a data group may contain not only data items but also other groups of data. 
+- Links to applicable NCTS code lists. Each link downloads a zip file from a separate non-HMRC site. The code lists are updated regularly. Consider setting up an automated way to download the files regularly and integrate them with your software.
+- Links to applicable rules and conditions. 
+
```

Comment:
Should we mention the header elements?  Something like...

- Header elements (XML fields that can contain other XML fields) are highlighted in *bold*

Reply:
@crystalmark - Good shout. To be fair, though, that change was agreed and implemented after Peter ran the review of the email text two days in Google Docs. Originally, I thought that you were going to show these formatting changes (agreed yesterday) at next week's TIS catch-up and get feedback. Do these documentation changes need to be live before then?

Reply:
Done.